libscanbuild/intercept.py

```diff
@@ -154,6 +154,36 @@ def intercept_compiler_wrapper(_, execution):
         logging.warning(message_prefix, 'io problem')
 
 
+def expand_cmd_with_response_files(cmd):
+    """ Expand's response file parameters into actual parameters
+
+    MSVC's cl and clang-cl has functionality to prevent too long command lines
+    by reading options from so called temporary "response" files. These files
+    are ascii encoded and can contain compiler and linker flags and/or
+    compilation units.
+
+    For example, QT's qmake generates nmake based makefiles where the response
+    file contains all compilation units. """
+
+    def is_response_file(param):
+        """ checks if the given command line argument is response file  """
+        return param[:1] == '@' and os.path.isfile(param[1:])
+
+    def from_response_file(filename):
+        """ read and return command line argument list from response file
+
+        can throw IOException but typically can as for now, its caught in
+        `intercept_compiler_wrapper` level.  """
+        with open(filename[1:], 'r') as f:
+            return [arg.strip() for arg in f.read().split()]
```

Comment:
shell_split used now .. Did you ever write more tests against it ? I have a recollection that i've tried to use it while back and noticed some issues in windows..  Maybe it would be good time now to extend those tests ?